SU-HSDO/suhumsci
# HSD8-89 Allowed second line for site name

# NOT READY FOR REVIEW

# Summary
- Added 2nd line site name option

# Needed By (Date)
- End of sprint - aug 30

# Urgency
- Low

# Steps to Test

1. checkout this branch
1. sync site `blt drupal:sync --site=archaeology --sync-files`
1. go to the basic site settings form: `/admin/config/system/site-information`
1. populate the "Site name line 2" text field with something of your choice
1. go to the homepage and validate your text appears in the header under the site name.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
